packages/gnome_calculator.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Gnome_calculator < Package
+class Gnome_calculator < Meson
   description 'GNOME desktop calculator'
   homepage 'https://wiki.gnome.org/Apps/Calculator'
   version '43.0.1'
```

Comment:
Update to [45.0.2](https://gitlab.gnome.org/GNOME/gnome-calculator/-/tags/45.0.2)?